### PR TITLE
Corrected input sequence

### PR DESCRIPTION
**Pull request by ernOho**
It seems the values in kernel.set_measurements(...)[simple_example.py, 36] were not added in the correct sequence.

### CODE REVIEW

File: kernel/td_kernel_dmvw.py [hunk 1]
```diff
@@ -168,7 +168,7 @@ def calculate_maps(self):
             self.importance_weight_concentration_map[low_index_x:high_index_x, low_index_y:high_index_y] += omega * self.measurement_concentrations[i]
 
         # alpha
-        self.confidence_map = 1 - np.exp(-self.importance_weight_map / (self.sigma_omega * self.sigma_omega))
+        self.confidence_map = 1 - np.exp(-(self.importance_weight_map)**2 / (self.sigma_omega * self.sigma_omega))
```
Comment (smueller18) [inline kernel/td_kernel_dmvw.py]: As pointed out in #1, the non-squared form is preferred using the algorithm with the wind component.

File: demo/simple_example.py [hunk 2]
```diff
@@ -33,7 +33,7 @@
 # call Kernel
 kernel = TDKernelDMVW(min_x, min_y, max_x, max_y, cell_size, kernel_size, wind_scale, time_scale,
                       low_confidence_calculation_zero=True, evaluation_radius=evaluation_radius)
-kernel.set_measurements(positions_x, positions_y, concentrations, wind_speeds, wind_directions, timestamps)
+kernel.set_measurements(positions_x, positions_y, concentrations, timestamps, wind_speeds, wind_directions)
```
Comment (smueller18) [inline demo/simple_example.py:36]: Good catch, this is definitely a bug.

**Comment by smueller18** [comment]
@ernOho please revert your changes for the confidence map and rename the MR. Then I can merge the corrected input sequence.

**Comment by ernOho** [comment]
Hi @smueller18
I should have reverted it and renamed the MR. If not I'll try again(a bit new to git sorry!)
kind regards

**Pull request merged**